implot_internal.h

```diff
@@ -261,6 +261,7 @@ enum ImPlotTimeFmt_ {              // default        [ 24 Hour Clock ]
     ImPlotTimeFmt_SUs,             // :29.428 552    [ :29.428 552  ]
     ImPlotTimeFmt_SMs,             // :29.428        [ :29.428      ]
     ImPlotTimeFmt_S,               // :29            [ :29          ]
+    ImPlotTimeFmt_MinSMs,          // 21:29.428      [ 21:29.428    ]    
```

Comment:
i notice that in description i forgot to start with ":"; correct preview of result should be: 
`// :21:29.428      [ :21:29.428    ]`